service-commands/src/commands/service-commands.json

```diff
@@ -145,7 +145,7 @@
   },
   "user-metadata-node": {
     "protocol": "http",
-    "host": "localhost",
+    "host": "cn-um_creator-node_1",
```

Comment:
will this work as localhost? I think it probably will work just fine considering the execution environment of mad dog (unless you're running mad-dog local against a remote box)

Reply:
yeah, this will work for localhost!

Reply:
per offline, this has already been done for other services (like discovery-provider)